invenio.cfg

```diff
@@ -515,11 +530,25 @@ COMMUNITIES_PERMISSION_POLICY = ZenodoCommunityPermissionPolicy
 ZENODO_RECORDS_UI_CITATIONS_ENABLE = True
 
 
-# Zenodo-RDM specific configs
-
-OPENAIRE_PORTAL_URL = 'https://explore.openaire.eu'
-"""URL to OpenAIRE portal."""
-
 METRICS_UPTIME_ROBOT_METRIC_IDS = {}
 METRICS_UPTIME_ROBOT_URL = "https://api.uptimerobot.com/v2/getMonitors"
 METRICS_UPTIME_ROBOT_API_KEY = None
+
+# OpenAIRE configs 
```

Comment:
These configs are duplicated from `zenodo_rdm.openaire.config`. The reasoning behind this was to have `openaire` as a stand-alone module inside zenodo-rdm. That way, we can easily move it to another place later on and it will require very minor changes.

These configs in `invenio.cfg` are for the instance, therefore they will be there even if `openaire` is moved to another repo / module.